build-aux/teleproxy.mk

```diff
@@ -23,7 +23,7 @@ $(tools/telepresence): CGO_ENABLED = 1
 endif
 $(tools/telepresence): $(abspath $@)
 	mkdir -p $(@D)
-	cd $(OSS_HOME) && curl -s --fail -L https://app.getambassador.io/download/tel2/$$(go env GOOS)/$$(go env GOARCH)/latest/telepresence -o $(abspath $@) && chmod a+x $(abspath $@)
+	cd $(OSS_HOME) && curl -s --fail -L https://app.getambassador.io/download/tel2/$$(go env GOOS)/$$(go env GOARCH)/2.6.6/telepresence -o $(abspath $@) && chmod a+x $(abspath $@)
```

Comment:
General question about the Telepresence pin: is there a specific reason why we don't use latest such as knowing that the newer versions don't work or that CI is expecting a certain set of features/syntaxes from Telepresence and commands may change in newer versions?

Reply:
As I said in the commit message: I just checked what the last version was that we saw work in Emissary 1.14 CI, which turned out to be Telepresence 2.6.6; at the time I didn't have any idea what had changed in newer Telepresence that would break it.  After communicating with Thomas: Yes, there was an intentional breaking change in Telepresence 2.7.0 (it now requires you to explicitly `telepresence helm install` it in to your cluster, rather than automatically doing that). So assuming that I don't think it's worth the effort to switch switch 1.14's machinery to use Tel 2.7, I suppose I could have pinned it to the latest 2.6, instead of 2.6.6, but... eh.